docs/source/examples/BSTLD.rst

```diff
@@ -167,10 +167,11 @@ You can get one by index.
    :start-after: """Read Dataset / get label"""
    :end-before: """"""
 
-There is only one label type in "BSTLD" dataset, which is ``box2d``. The annotation "RedLeft" is
-stored in :ref:`reference/label_format:Category` of :ref:`reference/label_format:Box2D`.
-The annotation "{'occluded': True}" is stored in :ref:`reference/label_format:Attributes`
-of :ref:`reference/label_format:Box2D`.
+There is only one label type in "BSTLD" dataset, which is ``box2d``. See
```

Comment:
Please remove "See :ref:`this page <reference/label_format:Box2D>` for more details about the
structure of Box2D." to the end of this paragraph.